docs/gallery/config.json

```
@@ -0,0 +1,3 @@
+{
+    "theme": "grid"
+}
```

> The build times did get a bit worse with this, it'd be really cool to at least save on the jupyter notebook generation. I guess making it optional is just a matter of adding a flag somewhere in DemoCards, or is it more complex than that?

This doesn't work right now, but would disable all notebook generation in future DemoCards release https://github.com/johnnychen94/DemoCards.jl/pull/98

```suggestion
{
    "theme": "grid",
    "properties": {
        "notebook": "false"
    }
}
```

---

Great! I'll wait for the next DemoCards release to add this

---

In case I didn't make it clear, it's harmless to commit this change as it won't take any effect or break the current workflow right now.

---

Well. Let me make a release for it

---

Ah, of course, I can just merge this patch here and it will get faster as soon as your release goes through. Thanks a lot for the feature and the release!